clearnode/custody.go

```diff
@@ -334,16 +334,16 @@ func (c *Custody) handleJoined(logger Logger, ev *nitrolite.CustodyJoined) {
 			return fmt.Errorf("DB error fetching asset: %w", err)
 		}
 
-		tokenAmount := decimal.NewFromBigInt(channel.Amount.BigInt(), -int32(asset.Decimals))
+		amount := decimal.NewFromBigInt(channel.RawAmount.BigInt(), -int32(asset.Decimals))
```

Comment:
I can see a lot of casting from `rawAmount` to `amount`, and hesitating about adding a `RawToFormatted(...)` and `FormattedToRaw(...)` cast methods. WDYT?